app/Http/Requests/UpdateEmailRequest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace App\Http\Requests;
+
+use App\Models\User;
+use Illuminate\Foundation\Http\FormRequest;
+
+class UpdateEmailRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        if ($this->user()->exists()) {
+            return true;
+        }
+        return false;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'email' => 'required|email|max:255|unique:users'
+            //
+        ];
+    }
+
+    public function commit() {
+
+        $this->user()->email = $this->input('email');
```

Comment:
change input to get (to comply with coding guidelines) so:
$this->input('email'); should become $this->get('email');